pkgs/development/compilers/purescript/purescript/default.nix

```
@@ -28,17 +28,17 @@ in stdenv.mkDerivation rec {
       then
       fetchurl {
         url = "https://github.com/${pname}/${pname}/releases/download/v${version}/macos-arm64.tar.gz";
-        sha256 = "16ci26pgrw0zmnyn1zj129y9624cqwzrhqglc8mgfg4k7rxvqy2a";
+        sha256 = "1pk6mkjy09qvh8lsygb5gb77i2fqwjzz8jdjkxlyzynp3wpkcjp7";
       }
       else
       fetchurl {
         url = "https://123.github.com/${pname}/${pname}/releases/download/v${version}/macos.tar.gz";
```

```suggestion
        url = "https://github.com/${pname}/${pname}/releases/download/v${version}/macos.tar.gz";
```
this is broken please change it to the above so it works again.

---

actually I just made the change, would be nice if some one else reviewed again.